.github/workflows/release-integration.yml

```diff
@@ -88,3 +93,15 @@ jobs:
             -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG \
             -t $DOCKER_IMAGE_NAME:latest \
             .
+  
+  notify-failure:
+    if: ${{ always() && failure() }}
+    needs: [docker-integration]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify failure via Slack
+        uses: archive/github-actions-slack@master
```

Comment:
Let's pin this action to @v2 instead to avoid breaking changes and use Renovate to get updates

Reply:
Done